src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    <p className="Person__partner">
+      {(person.sex === 'm' && person.isMarried) && (
+        `${person.partnerName} is my wife`
+      )}
+
+      {(person.sex === 'f' && person.isMarried) && (
+        `${person.partnerName} is my husband`
+      )}
```

Comment:
let's make your code more clear
```suggestion
     {person.isMarried && (
      person.sex === 'm' 
        ? `${person.partnerName} is my wife`
        : `${person.partnerName} is my husband`
     )}
```